lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -10,6 +10,18 @@
   }
 }
 
+.card__column {
+  .photo__actions, .caption {
```

Comment:
I think should we rename photo__actions to something else then -- right now its being used in photo, figure, and was about to be added to cards here.